gen/edge.ml

```diff
@@ -583,7 +583,7 @@ let fold_tedges f r =
   let iter_edges = Misc.fold_to_iter fold_edges
 
 
-  let t = Hashtbl.create 101
+  let t = Hashtbl.create 40000
```

Comment:
What is the prupose of this change? Does it affect the hash of created litmus tests?

Reply:
No it does not affect the functionality. This is the initial size of the hashtable. I notice there are a lot of entry in the hashtable in `AArch64` situation so I change to a larger initial size.